src/main/java/formflow/library/utils/UserFileMap.java

```diff
@@ -0,0 +1,64 @@
+package formflow.library.utils;
+
+import formflow.library.data.UserFile;
+import java.util.Map;
+import java.util.HashMap;
+import java.util.UUID;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * A class to contain the file mapping for the client side rendering of DropZone files.
+ * <p>
+ * Warning: This class will be serialized and sent to the client side. Do not include sensitive information in here that would
+ * then be shared with the client. Only include information that can be shared.
+ * </p>
+ */
+@Slf4j
+@Getter
+public class UserFileMap {
+
+  // flow -> inputName -> fileId -> file info
+  private Map<String, Map<String, Map<UUID, Map<String, String>>>> userFileMap;
+
```

Comment:
If you have any better ideas for this, I am all ears/eyes. But maybe we'd do that outside of this ticket.

Reply:
Do we want to abstract the Map<UUID, <Map<String, String>> part into just UserFile?

Reply:
I can try this out. I was hesitant to because this is what our file uploader uses to load files on the page and it's different from what's in the UserFile object. That can be adjusted though.  I'll try an experiment.

Reply:
I tried this out and am stuck on the thumbnail, which we take in, but don't store.  So if we pass back the UserFile, we won't have the thumbnail readily available with it.  
So then, do we start to store the thumbnail?  

I'd like to not do this right now but consider it, just outside of this ticket.

Reply:
That seems totally reasonable. Part of the reason why we opted to use base64 encoded strings for the thumbs was so we didn't have to store the actual files. This change doesn't feel super necessary but could be worth considering down the road for quality of life when working with these more complex objects since it can be hard to maintain a mental model with all the nesting.